src/settings/OrderNumberForm.js

```diff
@@ -0,0 +1,159 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'redux-form';
+
+import {
+  sortBy,
+  uniqBy,
+} from 'lodash';
+
+import stripesForm from '@folio/stripes/form';
+import {
+  Accordion,
+  AccordionSet,
+  Button,
+  Checkbox,
+  Col,
+  Headline,
+  MultiSelection,
+  Pane,
+  Row,
+} from '@folio/stripes/components';
+
+class OrderNumberForm extends Component {
+  static propTypes = {
+    label: PropTypes.object.isRequired,
+    handleSubmit: PropTypes.func.isRequired,
+    pristine: PropTypes.bool,
+    submitting: PropTypes.bool,
+    initialValues: PropTypes.object,
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      prefix: [],
```

Comment:
What is the reason of having this state? I think it could be managed by redux-form `Field`
```
                    <Field
                      name="tags"
                      component={MultiSelection}
                      emptyMessage="Empty message"
                      fullWidth
                      dataOptions={OPTIONS}
                      actions={actions}
                    />
```